README.md

```diff
@@ -35,7 +36,7 @@ Bundled packages:
 
 Pro tip: you can use these utilities in the builtin Terminal app under `/data/service/hnp`:
 
-![](./screenshot_hishell.jpg)
+![](./screenshot_hishell_cn.jpg)
```

Comment:
No, please `mv screenshot_hishell_cn.jpg screenshot_hishell.jpg` and change the relevant links instead.